packages/react-vapor/src/components/refresh/RefreshCallback.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {IReactVaporState} from '../../ReactVapor';
+import {callIfDefined} from '../../utils/FalsyValuesUtils';
+import {IDispatch} from '../../utils/ReduxUtils';
+import {RefreshCallBackActions} from './RefeshCallbackActions';
+import {RefreshStatus, RefreshStatusSelectors} from './RefreshCallbackReducer';
+
+export interface IRefreshCallbackOwnProps {
+    id: string;
+    time?: number;
+    incremental?: number;
+    intervalInMs?: number;
```

Comment:
I don't think `intervalInMs` needs to be a props refreshing once per second is kind of a standard for timers

Reply:
I dont know how flexible we want to be with this component. I put all value from Interval as props. IMO, I think we can keep delay only, Keep it simple for now. Good idea